ZooRoyal/coding-standard-source
# Change PHPStan configuration

* Remove "Just-in-Time" configuration
* Add dynamic configuration

## How to test?

* clone coding-standard-source
* switch to composer project you want to check
* run `<path-to-coding-standard-source-repo>/tests/run-coding-standard.sh sca:stan`
